pkgs/tools/package-management/nix-prefetch/default.nix

```diff
@@ -7,7 +7,7 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "nix-prefetch";
-  version = "0.3.1";
+  version = "0.4.0";
```

Comment:
Strange, the hash didn't change (I find this unlikely) :thinking: ?

Reply:
Oh... wow... that's bad. Updated it, thanks!